tests/test.sh

```diff
@@ -11,7 +11,7 @@ if [ -z ${NO_APT_INSTALL+x} ]; then
   sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
   sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io php
   if [ "$(uname -m)" = "aarch64" ]; then
-    sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends python3-dev ruby3.0 build-essential nodejs
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends python3-dev ruby build-essential nodejs
```

Comment:
I think this depends on the Ubuntu version? What version are you using?

On my ubuntu 22.04 I see that ruby3.0 needs ruby and ruby needs ruby3.0, how did you encounter this problem?

Reply:
Okay, in any case I think it's fine to change to `ruby`.